example/src/utils/index.tsx

```diff
@@ -53,7 +54,7 @@ export const networkClientKeyExtractor = (item: NetworkClientItem) => {
 const buildDefaultApiClientConfiguration = (
     headers: Record<string, string> = {},
     requestAdapterConfiguration: RequestAdapterConfiguration = {
-        bearerAuthTokenResponseHeader: "token",
+        bearerAuthTokenResponseHeader: "Token",
```

Comment:
Why the change in the case?

Reply:
@migbot Aren't header names case insensitive?

Reply:
Yup they should be case insentive I believe

Reply:
Weird. I'm ok leaving it like this, but perhaps have a chst with server team to fully understand it?